### ISSUE TITLE
Cocoapods + statically linked xcframework + default.metallib

### ISSUE THREAD

**Issue by molind** [CocoaPods/CocoaPods#10453]
I'm working on a framework called `GLMap` and we ship it as statically linked xcframework and dynamically linked xcframework.

When dynamically linked version is used - correct version of framework included into the Product. And correct `default.metallib` is used from that framework. No hacks required.

At the same time when statically linked framework is used - no sign of resources from the framework is included into the resulting app. And I have to include them manually using
```
  s.ios.resource = ['GLMap.xcframework/ios-arm64_i386_x86_64-simulator/GLMap.framework/default.metallib']
  s.macos.resource = ['GLMap.xcframework/macos-arm64_x86_64/GLMap.framework/default.metallib']
```

When we use `default.metallib` from **ios** in **ios simulator** it doesn't work. But it works when `default.metallib` from ios simulator is used for ios. 

Each of platorms uses it's own flags to build `default.metallib`. For examples different `-target` flags:
```
-target air64-apple-macos10.13
-target air64-apple-ios13.1-macabi
-target air64-apple-ios9.0-simulator
-target air64-apple-ios9.0
```
Could you propose any way to include correct `default.metallib` to it's target?

**Comment by acecilia** [comment]
Experiencing the same here:
* When integrating a prebuilt static xcframework, the resources are not handled automatically by cocoapods and they are not copied to the final target

The possible workaround as mentioned in this issue is to be specific about the resources in the podspec: `s.ios.resource = ['GLMap.xcframework/ios-arm64_i386_x86_64-simulator/GLMap.framework/default.metallib']`

The problem with that is that for that to work we are **assuming that the resources of each framework inside an xcframework are the same**, which is usually the case, but cant be warranted

**Comment by dnkoutso** [comment]
This might be an issue instead.

Do you have a sample app handy?

**Comment by dnkoutso** [comment]
Closing due to inactivity.

**Comment by kambala-decapitator** [comment]
I've faced a similar issue but related to standard resource bundle. I build static and dynamic framework using pod project and overriding `MACH_O_TYPE` on the command line, pod project is a dynamic framework. Resource bundle ends up inside the static framework as well, but when installing the framework in consumer project, resource bundle isn't copied to the app bundle (unlike dynamic framework case).

But in the end I had to move resource bundle from framework to another location anyway to solve the above as well as installing bundle with SPM. This also adds a small benefit of having only a single resource bundle in the repo instead of lots of copies inside each xcframework slice.
